src/styles/main.scss

```diff
@@ -1,3 +1,47 @@
+@import "./blocks/variables";
+@import "./blocks/header";
+@import "./blocks/card";
+@import "./blocks/stars";
+@import "./blocks/logo";
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
don't use * selector for resetting margins and padding. It's only acceptable for setting box-sizing. reset margins and paddings for particular elements, for example: 

```scss
h1,
h2,
p,
// other elements {
  margin: 0;
  padding: 0;
}
```

Reply:
Please pay attention to this comment